vtm/src/org/oscim/utils/GeoPointUtils.java

```diff
@@ -66,7 +66,7 @@ public static boolean contains(GeoPoint[] geoPoints, GeoPoint geoPoint) {
     }
 
     /**
-     * Returns the distance between the given segment and point.
+     * Returns the squareDistance between the given segment and point.
```

Comment:
Is this after an automatic refactoring of the IDE?

Reply:
It seems so. I'll fix it